tfhe/src/shortint/server_key/scalar_mul.rs

```diff
@@ -200,9 +203,7 @@ impl ServerKey {
     /// assert_eq!(3, clear);
     /// ```
     pub fn unchecked_scalar_mul_assign(&self, ct: &mut Ciphertext, scalar: u8) {
```

Comment:
doesn't complain about needless self ? 🤔

Reply:
Apparently not
I don't understand why

Reply:
I guess the lint has false negatives